client/packages/host/src/Admin/ElectronSettings.tsx

```diff
@@ -93,47 +98,73 @@ export const ElectronSettings = () => {
       <Typography variant="h5" color="primary" style={{ paddingBottom: 25 }}>
         {t('heading.barcode-scanners')}
       </Typography>
+
       <Setting
-        title={t('label.current')}
-        component={
-          !scanner ? (
-            <Box display="flex">
-              <AlertIcon color="error" />
-              <Typography
-                component="div"
-                sx={{
-                  alignItems: 'center',
-                  display: 'inline-flex',
-                  fontSize: '14px',
-                  paddingLeft: 1,
-                }}
-              >
-                {t('messages.no-scanners-found')}
-              </Typography>
-            </Box>
-          ) : (
-            <Scanner scanner={scanner} />
-          )
-        }
-      />
-      <Setting
-        title=""
+        title={'Scanner Type'}
```

Comment:
no changes to existing settings, just moved them behind `scannerType === 'usb_serial'`